### PR TITLE
Features/minicart

### PR DESCRIPTION
**Pull request by yurii-kozlov**

### CODE REVIEW

File: components/Minicart/MinicartItem.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { getTheRightProductNameLink } from 'helpers/getTheRightProductNameLink';
+import React, { ReactElement } from 'react';
+import Link from 'next/link';
```
Comment (TechCossack) [inline components/Minicart/MinicartItem.tsx:3]: pls, always sort imports as I said in the previous  PR)

File: styles/layout/Minicart/Minicart.module.scss [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  padding-left: 2.3rem;
+  padding-right: 2.3rem;
+}
+
+.title {
+  color: $colorBlack;
+  font-style: normal;
+  font-weight: $fontWeightSemiBold;
+  font-size: 1.8rem;
+  line-height: 140%;
```
Comment (TechCossack) [inline styles/layout/Minicart/Minicart.module.scss:40]: why 140%?)

Comment (yurii-kozlov) [reply styles/layout/Minicart/Minicart.module.scss:40]: > why 140%?)

that was indicated in the design template)

Comment (yurii-kozlov) [reply styles/layout/Minicart/Minicart.module.scss:40]: or should I write it in rems?

Comment (TechCossack) [reply styles/layout/Minicart/Minicart.module.scss:40]: yes) 140% it's for design, you should use rem or didn't use at all) in our case it's not necessary)

Comment (yurii-kozlov) [reply styles/layout/Minicart/Minicart.module.scss:40]: ohhh thank you)

**Pull request merged**